modules/microshift-adding-app-rpms-to-blueprint.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-adding-app-rpms-to-blueprint_{context}"]
+= Adding application RPMs to a blueprint
+
+.Prerequisites
+* You have root access to the host.
+* Workload or application RPMs exist in the `~/rpmbuild/RPMS` directory.
+
+To add application RPMs to a blueprint, you must create a local repository that Image Builder can use to create the ISO. With this procedure, the required container images for your workload can be pulled over the network.
+
+. Create a local RPM repository by running the following command:
++
+[source,terminal]
+----
+$ sudo createrepo ~/rpmbuild/RPMS/
+----
+
+. Give Image Builder access to the RPM repository by running the following command:
++
+[source,terminal]
+----
+$ sudo chmod a+rx ~
```

Comment:
This section is not complete. What we're trying to say here is that Image Builder should have access to all the files it needs. The problem with the HOME directory is that it is configured with `0700` permissions by default, so this command fixes the permissions. 
However, it does not guarantee that Image Builder will be able to access the rest of the files it needs. 
In summary, I would change it to a more generic statement, without giving the command.

Reply:
@ggiguash Is there a link to a RHEL doc (or ours in Install?) that a user can reference to ensure that Image Builder can access all of the files? I would like to point the user in the right direction.